00XX-hst-realignment.md

```diff
@@ -0,0 +1,116 @@
+# HIP ##: HST Realignment
+
```

Comment:
I can't seem to edit your branch to add the template items your missing, so just adding them here:

You should move `- Authors` to be first on the list.
Then:
```
- Start Date:
- Category: [Economic? Technical? Governance?]
- Original PR:
- Tracking Issue:
- Vote Requirements: veHNT
```